pkg/cmd/version/root.go

```diff
@@ -0,0 +1,112 @@
+package version
+
+import (
+	"encoding/json"
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	// Flags
+	outputFormat string
+)
+
+var VersionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print idpbuilder version and environment info",
+	Long:  "Print idpbulider version and environment info. This is useful in bug reports and CI.",
+	RunE:  version,
+}
+
+func init() {
+	VersionCmd.Flags().StringVarP(&outputFormat, "output", "o", "", `Print the idpbuilder version information in a given output format. Accepts "wide", "json", and "yaml".`)
+}
+
+var (
+	idpbuilderVersion = "unknown"
+	goVersion         = runtime.Version()
+	goOs              = runtime.GOOS
+	goArch            = runtime.GOARCH
```

Comment:
*Note:*
Using the `runtime` package here [like Kind does](https://github.com/kubernetes-sigs/kind/blob/f7e5c8e407269a6d5b7a6f86e6b3f9c13e9d8e9b/pkg/cmd/kind/version/version.go#L53)